pom.xml

```
@@ -7,10 +7,145 @@
     <groupId>org.example</groupId>
     <artifactId>cinema-app</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
+        <jdk.version>11</jdk.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
+        <maven.checkstyle.plugin.configLocation>
+            https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
```

```suggestion
            https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
```
create file for storing checkstyle rules locally in your project and write path to it here, Mate Academy shouldn't be mentioned in this project